src/helpers/contacts.js

```diff
@@ -126,3 +126,27 @@ export const harmonizeAndSortByFamilyNameGivenNameEmailCozyUrl = (
   ])
   return sortedData
 }
+
+/**
+ * Sort and rework contacts according to 'keepIndexFullNameAndDisplayNameUpToDate' service status
+ * @param {bool} hasServiceBeenLaunched - 'keepIndexFullNameAndDisplayNameUpToDate' service launch status
+ * @param {array} contactsWithIndexes - Contacts with indexes
+ * @param {array} contactsWithNoIndexes - Contacts without indexes
+ * @returns {array} Sorted and harmonized contacts
+ */
+export const reworkContacts = (
+  hasServiceBeenLaunched,
+  contactsWithIndexes,
+  contactsWithNoIndexes
+) => {
+  // hasServiceBeenLaunched will be deprecated when
+  // all instances have been migrated.
```

Comment:
Are we actually planning a migration for this ? Or do we plan to wait until a certain date?

Reply:
We plan to wait until a certain date

Reply:
We do not plan anything yet. We can't wait until a certain date since we're not sure the app will be updated at some point. 

We have two options: 
- 1/ a rundeck action that performs this migration on all our cozies 
- 2/ Keep this code for ever. The code is small enough to keep it imho

Reply:
By 'certain date' I meant the day all cozies instance was migrated... my bad it was confusing.

Reply:
I just saw the comment was already there before, sorry for bringing it up again. That being said, if we don't have a migration plan, and no set date to remove this code, imo we should remove the comment as well.